tests/components/honeywell/test_diagnostics.py

```diff
@@ -29,7 +29,7 @@ async def test_entry_diagnostics(
     await hass.config_entries.async_setup(config_entry.entry_id)
     await hass.async_block_till_done()
     assert config_entry.state is ConfigEntryState.LOADED
-    assert hass.states.async_entity_ids_count() == 6
+    assert hass.states.async_entity_ids_count() == 8
```

Comment:
Are we not only increasing by one entity?

Also the comments about the number of entities seems incorrect in many places.

Reply:
There are 2 devices in this test - so we get 2 more switches.
The comments may be incorrect, I will review.